components/kyma-environment-broker/internal/orchestration/handlers/orchestration_handler.go

```diff
@@ -203,6 +203,10 @@ func (h *orchestrationHandler) getOperation(w http.ResponseWriter, r *http.Reque
 		h.log.Errorf("while getting runtime state for upgrade operation %s: %v", operationID, err)
 	}
 
+	if upgradeState.ClusterSetup != nil {
+		upgradeState.KymaConfigFromClusterSetup()
```

Comment:
do we want to run this for upgrades not connected to any "orchestration" as well maybe?
https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/broker/instance_update.go

Reply:
it depends on whether other endpoints are showing KymaConfig in a response, like this one: https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/common/orchestration/dto.go#L147

Reply:
good point, I think they don't